src/hooks/useTxPendingStatuses.ts

```diff
@@ -85,6 +87,7 @@ const useTxPendingStatuses = (): void => {
           dispatch(
             setPendingTx({
               chainId,
+              safeAddress,
```

Comment:
This leads to errors if the event is emitted from a different Safe.

E.g.
Safe A executes a tx and it is processing
Then Safe B is opened.
Not Safe A's tx is executed and moves to "indexing". We would not set Safe B to pending.

I'll address this issue.